Formula/redis.rb

```diff
@@ -12,6 +12,13 @@ class Redis < Formula
     sha256 "be205c8cecbf8691db5f06b60e9a5dccbfe1e6bef2e1676fe0dbdf109b2fdf9f" => :high_sierra
   end
 
+  patch do
```

Comment:
>  `depends_on (line 22) should be put before patch (line 15)`

Reply:
Sounds great – will fix that and push updated PR (flattened)